Kernel/Heap/kmalloc.cpp

```diff
@@ -26,7 +26,7 @@ static constexpr size_t CHUNK_SIZE = 64;
 #endif
 static_assert(is_power_of_two(CHUNK_SIZE));
 
-static constexpr size_t INITIAL_KMALLOC_MEMORY_SIZE = 2 * MiB;
+static constexpr size_t INITIAL_KMALLOC_MEMORY_SIZE = 16 * MiB;
```

Comment:
Maybe cross-check the commit message again:
2MiB is not enough for which bitmap, and is the new limit truely 128**MiB**?

Reply:
Done